### PR TITLE
reduce eth_log calls from event log send event logs for activities #6540

### LINKED ISSUE #6540: We seem to be making much more calls to Infura with v3.62

### PR DESCRIPTION
**Pull request by oa-s**
Closes #6540

### CODE REVIEW

File: modules/AlphaWalletWeb3/AlphaWalletWeb3/Promises/Promise+Batching.swift [hunk 1]
```diff
@@ -10,7 +10,7 @@ import Foundation
 import PromiseKit
 
 public class JSONRPCrequestDispatcher {
-    public var MAX_WAIT_TIME: TimeInterval = 0.1
+    public var MAX_WAIT_TIME: TimeInterval = 0.5
```
Comment (hboon) [inline modules/AlphaWalletWeb3/AlphaWalletWeb3/Promises/Promise+Batching.swift:13]: Have you try measuring/logging to get a feel how many requests this cuts down to?

Comment (hboon) [reply modules/AlphaWalletWeb3/AlphaWalletWeb3/Promises/Promise+Batching.swift:13]: Also: https://github.com/AlphaWallet/alpha-wallet-ios/issues/6540#issuecomment-1498346374

**Comment by oa-s** [comment]
> 
> Took a quick look. Can you explain how it helps?

Yes, this pr fixes force updating event logs, current implementation has two types of updates, force, and .waitForCurrent, on initial call we make force update, as well as token script changes. But when tokens balance charges we make wait for current update. So current implementation have bug when we always send force update. This pr fixes that

**Review by hboon — APPROVED** [review]
Thanks

**Pull request merged**